CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 ### Fixed
 
 - When using the `sandbox.sh` script, always wait for `/app` frontend to be open on all nodes before marking the service as open (#3779).
+- Every leaf in the Merkle Tree now includes a claims digest (#3606).
```

Comment:
Should this be under `### Changed`? I wonder if this changelog entry should be phrased in terms of user-visible change here, i.e. receipts?

Reply:
I was thinking of it in terms of fixing the security issue, rather than the user-visible change, but you're right we can list both.